search-app/resources/templates/tool-base.html

```diff
@@ -11,6 +11,19 @@
 <div id="metadata-preview">
   <p>Fetching Tool...</p>
 </div>
+<!-- TODO we need to update a new version of EUI compliant with bootstrap-5 fix styling in the meantime -->
+<style>
```

Comment:
Theres an existing `Warning: Failed prop type: Invalid prop `tool.lastUpdatedDate` of type `string` supplied to `BC`, expected `object`.
` we should fix on the `edsc-metadata-plugin`